src/test/java/duke/task/EventTest.java

```diff
@@ -0,0 +1,15 @@
+package duke.task;
+import java.time.LocalDate;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+public class EventTest {
+    @Test
+    public void testToStringWithLocalDateRange() {
```

Comment:
While the 2nd and 3rd part of test method names are optional, given this test name is quite long, perhaps can try to fit into 2-3 parts and separate the parts with underscores for readability